python/hail/docs/tutorials/09-matrixtable.ipynb

```diff
@@ -0,0 +1,490 @@
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "Some operations are unique to `MatrixTable`:\n",
+    "- The row fields can be accessed as a `Table` with [rows](https://hail.is/docs/devel/hail.MatrixTable.html#hail.MatrixTable.rows)\n",
+    "- The column fields can be accessed as a `Table` with [cols](https://hail.is/docs/devel/hail.MatrixTable.html#hail.MatrixTable.cols).\n",
+    "- The entire field space of a `MatrixTable` can be accessed as a coordinate-form `Table` with [entries](https://hail.is/docs/devel/hail.MatrixTable.html#hail.MatrixTable.entries). Be careful with this! While it's fast to aggregate or query, trying to write this `Table` to disk could produce files _thousands of times larger_ than the corresponding `MatrixTable`.\n",
+    "\n",
+    "Let's explore `mt` using these tools.  Let's get the size of the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "mt.count() # (rows, cols)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "Let's look at the first few row keys (variants) and column keys (sample IDs)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "mt.rows().select('locus', 'alleles').show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "mt.s.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "Let's investigate the genotypes and the call rate.  Let's look at the first few genotypes:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "mt.GT.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "All homozygous reference, which is not surprising.  Let's look at the distribution of genotype calls:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "mt.aggregate_entries(hl.agg.counter(mt.GT.n_alt_alleles()))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "Let's compute the call rate directly, and then plot the distribution of call rates per vairant."
```

Comment:
Let's compute the overall call rate directly, and then plot the distribution of call rate per variant.